source/project_schema.ts

```diff
@@ -18,8 +18,8 @@ import type { Data, QueryResponse } from "./types.js";
  *
  * @memberof project_schema
  */
-export function getStatuses(
-  session: Session,
+export function getStatuses<TEntityTypeMap extends Record<string, any>>(
+  session: Session<TEntityTypeMap>,
```

Comment:
this should automatically get the entitytypemap inferred from the provided session, make sure it works so you don't have to specify it